react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Hotbutton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicks: 0 };
```

Comment:
Any properties you plan to use in state should have a starting value, so since you are saving the `className` in state it should have a starting value here:

```JSX
this.state = {
  clicks: 0,
  className: ''
};
```